<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtGuiFileTestClass</class>
 <widget class="QMainWindow" name="QtGuiFileTestClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>989</width>
    <height>654</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtGuiFileTest</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="result">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>缓冲区</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>461</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="timeFre">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>20</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>时频切换</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addNoise">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>20</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>加噪</string>
    </property>
   </widget>
   <widget class="QPushButton" name="filtering">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>70</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>滤波</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openFile">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>信号输入</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>989</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtGuiFileTest.qrc"/>
 </resources>
 <connections/>
</ui>
